src/features/admin/pages/AdminStatistics.css

```
@@ -0,0 +1,140 @@
+  top: 110%;
+  left: calc(100% + 8px);
+
+  width: 160px;
+
+  background: white;
+  border: 1px solid #ddd;
+  border-radius: 8px;
+
+  display: block;
+
+  position: absolute;
```

Biome (2.4.6)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

(lint/suspicious/noDuplicateProperties)